### ISSUE TITLE
multi-label classifier supported eg. "0_man_person.jpeg"

### ISSUE THREAD

**Issue by LukeAI** [AlexeyAB/darknet#5789]
Can a classifier be trained to predict multiple classes per image?

Since the output of a classifier is a softmax, multiple labels per image should be supported?

eg. if my classes are: person, man, woman, dog, cat
Can I label my data like:
```
0_man_person.jpeg
1_woman_person.jpeg
2_cat.jpeg
```

and then train a classifier in the standard way that will correctly give me high probabilities for both "man" and "person" when it sees a picture of a man?

**Comment by Amore-HDU** [comment]
Hello,
Is you dataset overlap seriously?

**Comment by LukeAI** [comment]
> Hello,
> Is you dataset overlap seriously?
yes, I need a classifier for a dataset with multiple labels per image. Not the exact example I gave.

**Comment by LukeAI** [comment]
@AlexeyAB  sorry to bother you, I wonder if you could let me know?

**Comment by Maioy97** [comment]
@LukeAI did you find out how to do it ?

**Comment by LukeAI** [comment]
I think it's not possible. maybe just don't use darknet if you need this. I ended up using pytorch.
